internal/lib/container_server.go

```diff
@@ -272,6 +243,44 @@ func (c *ContainerServer) LoadSandbox(id string) (retErr error) {
 	if err != nil {
 		return err
 	}
+
+	if err := sb.SetInfraContainer(scontainer); err != nil {
+		return err
```

Comment:
Let’s wrap the errors to add context to it:
```suggestion
		return errors.Wrap(err, "set intra container")
```